sky/utils/cli_utils/cost_utils.py

```diff
@@ -0,0 +1,123 @@
+"""Utilities for sky cost report and sky spot cost."""
+from typing import Any, Dict, List
+import colorama
+from sky import global_user_state
+
+
+def get_cost_report(cluster_status):
+    cost = cluster_status['total_cost']
+
+    if not cost:
+        return '-'
+
+    return f'${cost:.3f}'
+
+
+def get_status_for_cost_report(cluster_status):
+    status = None
+    if 'status' in cluster_status:
+        status = cluster_status['status']
+
+    if status is None:
+        return f'{colorama.Style.DIM}{"TERMINATED"}{colorama.Style.RESET_ALL}'
+    return status.colored_str()
+
+
+def get_resources_for_cost_report(cluster_status):
+    launched_nodes = cluster_status['num_nodes']
+    launched_resources = cluster_status['resources']
+
+    launched_resource_str = str(launched_resources)
+    resources_str = (f'{launched_nodes}x '
+                     f'{launched_resource_str}')
+
+    return resources_str
+
+
+def aggregate_all_records(verbose: bool) -> List[Dict[str, Any]]:
+    rows = global_user_state.get_distinct_cluster_names_from_history()
+    records = global_user_state.get_clusters_from_history()
+
+    agg_records: List[Dict[str, Any]] = []
+
+    for (cluster_name,) in rows:
+        if verbose:
+            agg_records += get_split_view_records_by_name(cluster_name, records)
```

Comment:
should we keep the term consistent across different functions? We seems to have three different terms: `condense`, `split` and `verbose`